keda/templates/NOTES.txt

```diff
@@ -20,8 +37,6 @@ Get details about a deployed ScaledObject:
 Get an overview of the Horizontal Pod Autoscalers (HPA) that KEDA is using behind the scenes:
   kubectl get hpa [--all-namespaces] [--namespace <namespace>]
 
--------------------------------------------------------------------------------------
```

Comment:
The main reason why I added this is if there are any warning printed, they will be right after the line above, ie there is not a empty line printed. And it is not very clear.

Reply:
Should we undo this?

Reply:
Yes, but let's only show the line if there is an effective warning then. If not, this is just pointless imo

Reply:
Updated